ch_backup/config.py

```diff
@@ -153,64 +162,25 @@ def _as_seconds(t: str) -> int:
         "ca_bundle": [],
         "disable_ssl_warnings": False,
     },
-    "logging": {
-        "version": 1,
-        "formatters": {
-            "ch-backup": {
-                "format": "%(asctime)s %(processName)-11s %(process)-5d [%(levelname)-8s] %(name)s: %(message)s",
-            },
-            "boto": {
-                "format": "%(asctime)s %(processName)-11s %(process)-5d [%(levelname)-8s] %(name)s: %(message)s",
-            },
-        },
-        "handlers": {
-            "ch-backup": {
-                "class": "logging.FileHandler",
-                "filename": "/var/log/ch-backup/ch-backup.log",
-                "formatter": "ch-backup",
-            },
-            "boto": {
-                "class": "logging.FileHandler",
-                "filename": "/var/log/ch-backup/boto.log",
-                "formatter": "boto",
-            },
-            "clickhouse-disks": {
-                "class": "logging.FileHandler",
-                "filename": "/var/log/ch-backup/clickhouse-disks.log",
-                "formatter": "ch-backup",
-            },
-        },
-        "loggers": {
-            "ch-backup": {
-                "handlers": ["ch-backup"],
-                "level": "DEBUG",
-            },
-            "botocore": {
-                "handlers": ["boto"],
-                "level": "INFO",
-            },
-            "botocore.endpoint": {
-                "level": "DEBUG",
-            },
-            "botocore.vendored.requests": {
-                "level": "DEBUG",
-            },
-            "botocore.parsers": {
-                "level": "DEBUG",
-            },
-            "urllib3.connectionpool": {
-                "handlers": ["boto"],
-                "level": "DEBUG",
-            },
-            "clickhouse-disks": {
-                "handlers": ["clickhouse-disks"],
-                "level": "INFO",
-            },
-            "zookeeper": {
-                "handlers": ["ch-backup"],
-                "level": "DEBUG",
-            },
-        },
+    "loguru": {
```

Comment:
I'd propose to change the config structure.
I propose to move format definition in a separate key instead of function `_handler_configuration`
Like so:
```
"loguru" : {
  "formats": {
    "ch-backup": "{time:YYYY-MM-DD H:m:s,SSS} {process.name:11} {process.id:5} [{level:8}] {extra[logger_name]}: {message}",
  },
  "handlers": {
    "ch-backup": {
      "sink": "/var/log/ch-backup/ch-backup.log",
      "level": "DEBUG",
      "format": "ch-backup"
      # "enqueue": True  # don't define this but set all handlers in code as enqueued without option
    } 
    #...
  } 
  #...
}
```